.github/workflows/backend-tests.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       matrix:
         # If we start actively supporting other databases, they should be added here so that we can test against them.
-        testdatabase: ["mysql:5.7.44", "mysql:8.0.36", "mariadb:10.3.39"]
+        testdatabase: ["mysql:8.0.36", "mariadb:10.3.39"]
```

Comment:
I figured I'd make this a PR as a reminder for where this config lives. It's really easy to add/remove db versions to test against so if we upgrade in the future, all we need to do is remember to update this list